### PR TITLE
Add CTO & COO to the Website

### PR DESCRIPTION
**Pull request by dvvanessastoiber**
Dominic and Anita are added to the Website. Furthermore, all links to social media profiles etc. are now opened in a new tab and the datavisyn linkedin profile is added to the social buttons in the footer.

### CODE REVIEW

**Comment by dvvanessastoiber** [comment]
Anita and Dominic are added to the Website:

image

Linkedin Profile added to social buttons:
image

**Comment by mstreit** [comment]
Looks good. The heading should be changed from 'Founders' to 'Founders and Management'. And maybe a 3x2 layout looks better?

**Comment by mstreit** [comment]
And maybe we should use the full job title instead of the abbreviations? (Chief Technology Officer instead of CTO, etc.).

**Comment by dvvanessastoiber** [comment]
Is it okay like that?

image

**Comment by mstreit** [comment]
Perfect, thanks! Please also check if it looks okay on mobile devices and different resolutions/window sizes.

**Comment by lehnerchristian** [comment]
here are some more screenshots

the smartphone version:
website_smartphone

the tablet version (ipad in this case):
website_ipad_3x2


the tablet version looks probably a bit crowded with a 3x2 grid (this was already a problem before). alternatively we could use a 2x3 grid:
website_ipad_2x3

**Comment by lehnerchristian** [comment]
and the header is now `Founders and Management` --> should we update the link in the navigation?

**Comment by mstreit** [comment]
The term 'Team' is probably better suited for the top menu. Thanks!

File: _sass/team.scss [hunk 1]
```diff
@@ -12,12 +12,12 @@
 
 @media screen and (min-width: 768px) {
   .team .team-member {
-    width: 33%;
+    width: 35%;
```
Comment (lehnerchristian) [inline _sass/team.scss]:
```suggestion
    width: 50%;
```

let's use 50%, because marc's long subtitle will break lines multiple times

**Review by lehnerchristian — CHANGES_REQUESTED** [review]

**Review by lehnerchristian — APPROVED** [review]

**Pull request merged**